### PR TITLE
Added support to read outside error count returned from XML formatter.

### PR DESCRIPTION
**Pull request by pingfr**
I opened a issue #85 to get error count from errors outside of an example.  This is the pull request for that issue.  Please let me know if there is any further action needed.

### CODE REVIEW

File: lib/rspec_junit_formatter/rspec3.rb [hunk 1]
```diff
@@ -45,6 +45,10 @@ def examples
     @examples_notification.notifications
   end
 
+  def error_count
+    reports_errors_outside? ? @summary_notification.errors_outside_of_examples_count : 0
```
Comment (sj26) [inline lib/rspec_junit_formatter/rspec3.rb]: How about:

```suggestion
    # Introduced in rspec 3.6
    if @summary_notification.respond_to?(:errors_outside_of_examples_count)
      @summary_notification.errors_outside_of_examples_count
    else
      0
    end
```

**Comment by sj26** [comment]
I think you'll need to add an `errors_count` method within `lib/rspec_junit_formatter/rspec2.rb` to fix the cross version tests.

They can be run with:

```
bundle
bundle exec appraisal install
bundle exec appraisal rspec
```

You can test just rspec 2.x with:

```
bundle exec appraisal rspec-2-x rspec
```

File: lib/rspec_junit_formatter/rspec3.rb (hunk 1 above)
Comment (pingfr) [reply lib/rspec_junit_formatter/rspec3.rb]: That works for me.  I will make that change and push it.

**Comment by pingfr** [comment]
Travis is happy now.  Let me know if there are any other concerns before you accept the PR.  Also wondering when it will be part of an official release.

**Pull request merged**